src/mem/cache/prefetch/signature_path_v2.cc

```diff
@@ -125,7 +125,8 @@ SignaturePathV2::handlePageCrossingLookahead(signature_t signature,
     GlobalHistoryEntry *gh_entry = globalHistoryRegister.findVictim(0);
     assert(gh_entry != nullptr);
     // Any address value works, as it is never used
-    globalHistoryRegister.insertEntry(0, gh_entry);
+    bool is_secure = false;
```

Comment:
Since the `is_secure` variable is not changed afterwards, can you change this to `constexpr bool is_secure = false;`?